### PR TITLE
Add Power BI as client and TCB 61

### PR DESCRIPTION
**Pull request by mosabua**
Fyi @pichlerpa .. please let me know of any other additions or changes.

### CODE REVIEW

**Comment by pichlerpa** [comment]
Looks good! Nothing to add from my side. Looking forward to the call. Patrick

**Comment by mosabua** [comment]
@colebow .. this is ready for review apart from the timestamps that I am now working on.

File: _episodes/61.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+## PowerBI overview
+
+[Power BI](https://powerbi.microsoft.com/) is an interactive data visualization
+software product suite developed by Microsoft with a primary focus on business
+intelligence. We talk about the different available products and features, and
+their usage in the community.
+
+## Trino client support options for Power BI
+
+Typically Power BI relies on ODBC drivers for connecting to specific data
```
Comment (colebow) [inline _episodes/61.md:92]:
```suggestion
Typically, Power BI relies on ODBC drivers for connecting to specific data
```

**Review by colebow — APPROVED** [review]
one nit but LGTM

**Pull request merged**